src/exception/PositionNotFoundException.kt

```diff
@@ -0,0 +1,7 @@
+package exception
+
+class PositionNotFoundException(override val message: String = NO_SUCH_COORDINATE_EXCEPTION_MESSAGE) : InvalidMoveException(message) {
+    companion object {
+        const val NO_SUCH_COORDINATE_EXCEPTION_MESSAGE: String = "The provided coordinate is not valid."
+    }
```

Comment:
minor: I feel like this is an overuse/abuse of companion objects. Kotlin's companion objects are not really an alias for java's static. To do a static, just declare a 'val <var>' in a global scope (ie before your class declaration). For strings, you can also do a 'const val <strFieldName> = "..."'. Or, not even pull it out into a constant at all, because why, if you're only going to reference it once?

More info on companion object's intended use: https://discuss.kotlinlang.org/t/what-is-the-advantage-of-companion-object-vs-static-keyword/4034/4

Reply:
alright, thanks for the suggestion